authority/provisioner/sign_ssh_options.go

```diff
@@ -381,25 +386,25 @@ type sshCertDefaultValidator struct{}
 func (v *sshCertDefaultValidator) Valid(cert *ssh.Certificate, o SignSSHOptions) error {
 	switch {
 	case len(cert.Nonce) == 0:
-		return errors.New("ssh certificate nonce cannot be empty")
```

Comment:
Feels like these should all be badrequest

Reply:
This is tricky, and perhaps it should be 500, it's a default validator that all our provisioners do to make sure a certificate is properly formatted.